plaso/storage/file_interface.py

```diff
@@ -0,0 +1,1147 @@
+# -*- coding: utf-8 -*-
+"""Storage interface classes for file-backed stores."""
+
+from __future__ import unicode_literals
+
+import shutil
+
+import abc
+import os
+import tempfile
+
+from plaso.lib import definitions
+from plaso.serializer import json_serializer
+from plaso.storage import interface
+
+class SerializedAttributeContainerList(object):
+  """Serialized attribute container list.
+
+  The list is unsorted and pops attribute containers in the same order as
+  pushed to preserve order.
+
+  The GetAttributeContainerByIndex method should be used to read attribute
+  containers from the list while it being filled.
+
+  Attributes:
+    data_size (int): total data size of the serialized attribute containers
+        on the list.
+    next_sequence_number (int): next attribute container sequence number.
+  """
+
+  def __init__(self):
+    """Initializes a serialized attribute container list."""
+    super(SerializedAttributeContainerList, self).__init__()
+    self._list = []
```

Comment:
what is the purpose of `_list` ?

Reply:
This class was just moved, but there doesn't seem to be another list that this would get confused with other than the class itself.